src/components/desktop/desktop-aside.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="x-background--neutral-100 x-list__item--expand x-list x-list--vertical">
+    <div
+      class="
+        x-list x-list--horizontal x-list--align-center
+        x-border-width--bottom-01
+        x-border-color--neutral-10
+        x-padding--08 x-padding--top-06 x-padding--bottom-06
+      "
+    >
+      <span class="x-title2 x-font-color--neutral-10 x-font-weight--bold x-margin--right-auto">
+        {{ $t('facetsPanel.title') }}
+      </span>
+      <BaseIdModalClose class="x-button--ghost x-button--ghost-end" modalId="right-aside">
+        <CrossTinyIcon class="x-icon--l x-font-color--neutral-10" />
+      </BaseIdModalClose>
+    </div>
+    <div class="x-scroll x-list__item--expand x-padding--08">
+      <Sort />
+      <CustomFacets
+        v-if="$x.totalResults > 0"
+        class="x-list--gap-06 x-list--padding-05 x-list--padding-top"
```

Comment:
These classes can be deleted if the facets headers spacings are set using the padding token.